services/apm/apm.service.js

```diff
@@ -65,6 +56,16 @@ class APMDownloads extends BaseAPMService {
       capture: ['repo'],
     }
   }
+
+  static get examples() {
+    return [
+      {
+        exampleUrl: 'PACKAGE',
```

Comment:
What do you think about using Sinatra-inspired syntax?

`:package`

This is a bit aspirational: I did a little work a while back with a library that took that kind of pattern and automatically converted it to regex, and also allowed you to substitute specific values into the pattern. That kind of thing would be really useful on the frontend.

It's also a format that's very recognizable, and attractive.

Reply:
Yeah. I also thought about `{param}` as that is used by a few frameworks, but I went with the `PARAM` style as we've done that in some other places. I'm down with`:param`.